desktop/install/linux-install.md

```diff
@@ -268,6 +270,17 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the default (and currently only) mechanism to enable file sharing between the host and Docker Desktop VM. In order not to require elevated privileges, without unnecessarily restricting operations on the shared files, Docker Desktop runs the file sharing service (`virtiofsd`) inside a [user namespace](https://man7.org/linux/man-pages/man7/user_namespaces.7.html) with UID and GID mapping configured. As a result Docker Desktop will rely on the host to be configured to enable the current user to use subordinate ID delegation. For this to be true [/etc/subuid](https://man7.org/linux/man-pages/man5/subuid.5.html) and [/etc/subgid](https://man7.org/linux/man-pages/man5/subgid.5.html) have to be present (Docker Desktop currently only supports subordinate ID delegation configured via files). In case of both Docker Desktop will use the first entry corresponding to the current user, to set up mappings for IDs above 0, and map the current users ID and GID to 0.
+
+In case `/etc/subuid` and `/etc/subgid` are missing they will need to be created. Both should contain entries in the following form `<username>:<start of id range>:<id range size>`. E.g. to allow a user whose username is `docker` to use IDs from 100000 to 165535:
```

Comment:
might be good to use something like `myusername` instead of `docker` for the examples; using `docker` is easy to be confused for "this must be `docker`" (some special username), whereas `myusername` is more verbose on "fill in your username here"

Reply:
Good point, I was wondering what would be a good example name to use. I usually go with putting stuff to fill in in angle brackets, but lately I've ran into cases where that confused someone.

Reply:
Yes, it remains tricky; indeed, use `<FOO>`, and users may try it literally (shell complaining about piping). Use something `ALLCAPS` and users may now think it must be uppercase (and/or it's case-insensitive).

We need to come with some conventions, and document them somewhere in the docs ("here's the conventions we use (describe what terms, and how we use example/placeholder values).

Reply:
Yeah, that sounds like a good idea. It could also help avoid some confusing situations when doing support :)